### DISCUSSION TITLE
Need help with a small confusion

### DISCUSSION THREAD

**Discussion by aaashir** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#38] [General]
Hello!

I have Dell Latitude E7470, I just saw your repo and it's well maintained, thumbs up to you!

But I have a question, Do I need a different EFI config for Intel Core i5-6300U with SAMSUNG SSD and with Dell Touchpad. What I am saying is I have the same laptop but my processor is **Intel Core i5-6300U** and SSD is **SAMSUNG** with **Dell Touchpad**. Will this EFI work for me or if not? Can anyone help with the clarification?

Thanks in advance! :)

**Comment by peretm** [comment]
I have exactly the same, and in installation, it downloads the software but after the first restart, I choose install big sur and it keeps restarting...
Any help will be apreciated.

**Reply by aaashir** [reply]
@peretm - Yes same. It's a 12 GB Download.

**Reply by peretm** [reply]
@aaashir check this out: https://osxlatitude.com/forums/topic/9179-dell-latitude-e7x70-installation-guides-uefi-only-clover-and-opencore/?do=findComment&comment=104256

**Reply by aaashir** [reply]
> @aaashir check this out: https://osxlatitude.com/forums/topic/9179-dell-latitude-e7x70-installation-guides-uefi-only-clover-and-opencore/?do=findComment&comment=104256

@peretm - They haven't included the links to download the dmg or full macOS.

**Reply by peretm** [reply]
i followed this tutorial. Basically you prepare a install disc of Big Sur 11.4 on a pen drive, from a real mac and then change the EFI folder for the one in the osxlatitude forum, modify bios setting as in the osxlatitude forum and boot the pen drive. Install and enjoy.

**Reply by peretm** [reply]
https://www.freecodecamp.org/news/build-a-hackintosh/

**Reply by aaashir** [reply]
@peretm - Is yours working fine?

**Reply by peretm** [reply]
yes. working as a charm
![Captura de Pantalla 2021-06-27 a les 23 54 35](https://user-images.githubusercontent.com/86209247/123560555-35f7b780-d7a3-11eb-918a-7786c8c6ee62.png)

**Reply by aaashir** [reply]
I don't have access to Mac. Can you share the Big Sur 11.4 image?

**Reply by aaashir** [reply]
@peretm - I just downloaded the Big Sur 11.4. Can you share the EFI folder as I can't able to download it from osxlatitude?

**Comment by adityabakare** [comment]
> Hello!
> 
> I have Dell Latitude E7470, I just saw your repo and it's well maintained, thumbs up to you!
> 
> But I have a question, Do I need a different EFI config for Intel Core i5-6300U with SAMSUNG SSD and with Dell Touchpad. What I am saying is I have the same laptop but my processor is **Intel Core i5-6300U** and SSD is **SAMSUNG** with **Dell Touchpad**. Will this EFI work for me or if not? Can anyone help with the clarification?
> 
> Thanks in advance! :)

Yes, it should work with the i5 model too, but in the previous update, I think I made some changes which were unfavorable, so probably it isn't working.

Sorry for the late reply. I was away for a while. I guess I need to update the EFI ASAP

**Reply by aaashir** [reply]
Hi @adityabakare!

Please do that, it'll be very helpful. By the way, I just found an EFI on OSXLatitude.com for my exact model.

Link: https://osxlatitude.com/forums/topic/9179-dell-latitude-e7x70-installation-guides-uefi-only-clover-and-opencore/?do=findComment&comment=104256

Can you check and let me know if it's okay?

Thanks!

**Reply by adityabakare** [reply]
The OSXLatitude EFI should work too. But they use a lot of outdated Hotpatches, which I've avoided in my releases. My EFI is somewhat inspired from them, so using it should be fine too.
